pre/remap_restart/remap_catchANDcn.py

```diff
@@ -59,6 +59,9 @@ def remap(self):
      surflay        = config['output']['surface']['surflay']
      in_tilefile    = config['input']['surface']['catch_tilefile']
 
+     job = 'SLURM'
```

Comment:
Ooh. We might want a better way of doing this. I can think of a couple things.

First, we can use `configure_file` to do this for us as we have variables that will have `NCCS` or `NAS` as the output. So you could use, say, `site = @GEOS_SITE@` and then say if NCCS, slurm, else PBS?

Reply:
The "bc_base" has the final say if it is PBS or SLURM . And bc_base is derived from the site. So I think it is ok to assign an initial value 'SLURM' here.

Reply:
changed!